### DISCUSSION TITLE
Possibilities of importing 2 phase voxel based synthetic microstructure

### DISCUSSION THREAD

**Discussion by hem-28** [BlueQuartzSoftware/DREAM3D#992] [Q&A]
Hi,
Here is my problem description, any suggestions would be great :
1. We have an internal workflow, which generates a voxel based microstructure from LOM and/or EBSD images. This workflow makes sure that the morphology (the spatial distribution of each phase and its shape) of each phase is statistically represented. In the end I obtain an Abaqus .inp file and a .vtk with RECTILINEAR_GRID data and look up table for phase IDs.  For further processing of the synthetic microstructure, I am looking for ways to import this in to DREAM3D. I could only find Structured points data import filter. Would there be workaround to import my data , either .inp or rectilinear grid. 
2. If I succeed importing my voxel data, I would want to assign a grain size distribution to each of my phases and enforce a texture. Would something like this be possible? 
3. I did not find a filter in DREAM3D which could control the spatial distribution of phase. Am I wrong here? 

Thanks!

**Update:**
I figured out how to convert the RECTILINEAR_GRID data to STRUCTURED_POINTS.  Suggestions on the other points would be great.

**Comment by imikejackson** [comment]
Essentially you are generating a 3D microstructure, you want to import that microstructure into DREAM.3D and generate new microstructures based on calculated grain size distributions for each phase and also enforce a crystallographic texture on the newly generated microstructure?

**Reply by hem-28** [reply]
@imikejackson Yes exactly. Basically add grain size distribution and crystallographic texture to the old microstructure. The important aspect here would be to retain the spatial distribution of phases.
